modules/twinkleunlink.js

```diff
@@ -164,7 +164,8 @@ Twinkle.unlink.callbacks = {
 					apiobj.params.form.append( { type:'header', label: 'File usage' } );
 					namespaces = [];
 					$.each(Twinkle.getPref('unlinkNamespaces'), function(k, v) {
-						namespaces.push(Morebits.wikipedia.namespacesFriendly[v]);
+						var nsName = mw.config.get('wgFormattedNamespaces')[v];
```

Comment:
This might just be me, and it's certainly nitpicking, but I see:
````
var = value;
array.push(ternary)
````
As more complicated than both
````
var = ternary
array.push(var)
````
and
````
array.push(ternary)
````

Reply:
just wanted to avoid writing `mw.config.get('wgFormattedNamespaces')` twice!

Reply:
Why twice?  `v` holds the number, we only care if `v === 0`, right?  So wouldn't something like either
```suggestion
						namespace.push(v === 0 ? '(Article)' : mw.config.get('wgFormattedNamespaces')[v];
```
or
```suggestion
						var nsName = (v === 0 ? '(Article)' : mw.config.get('wgFormattedNamespaces')[v]);
						namespaces.push(nsName);
```
do the trick?

Reply:
Oh yeah, that's neat. Didn't think of testing on `v`. It's been a long night...